
try {
 
} catch (CustomThrowable e) {
 
}
//----------------------------
try {
 
} catch (CustomException e) {
 
}